README.md

```diff
@@ -1051,6 +1055,10 @@ Some things to note about activity definitions:
   activity may need (e.g. a DB connection). The instance method should be what is registered with the worker.
 * Activities can also be defined on callable classes (i.e. classes with `__call__`). An instance of the class should be
   what is registered with the worker.
+* The `@activity.defn` can have `dynamic=True` set which means all otherwise unhandled activities fall through to this.
+  If present, cannot have `name` argument, and the activity function must accept a single parameter of
+  `Sequence[temporalio.common.RawValue]`. The payload of the raw value can be converted via
+  `activity.payload_converter().from_payload`.
```

Comment:
Same, is this also missing `name`?

Reply:
We surface this via contextual activity info, so I intentionally leave it off here (same as .NET)